html/user.hbs

```diff
@@ -1,6 +1,9 @@
-<script src="/scripts/share.js" type="module"></script>
+<script src="/scripts/auth.js" type="module"></script>
```

Comment:
does share.js need to be included somewhere else?

Reply:
no that was accidentally deleted. It needs to be on the user.hbs